Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM ruby:3.2.2-alpine
+
+ENV BUNDLER_VERSION=2.4.20
+
+RUN apk add --update --no-cache \
+      binutils-gold \
```

Comment:
do you actually need all of this stuff?

Reply:
I think because we're using `alpine`, we do -- i followed a guide for this setup.